locations/spiders/monsoon_gb.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+import scrapy
+from locations.linked_data_parser import LinkedDataParser
+
+
+def from_wikidata(name, code):
+    return {"brand": name, "brand_wikidata": code}
+
+
+class MonsoonGBSpider(scrapy.spiders.SitemapSpider):
+    name = "monsoon_gb"
+    sitemap_urls = ["https://stores.monsoon.co.uk/sitemap.xml"]
+    download_delay = 0.2
+    MONSOON = from_wikidata("Monsoon", "Q65007501")
+    ACCESSORIZE = from_wikidata("Accessorize", "Q65007482")
+    item_attributes = {"country": "GB"}
+
+    def parse(self, response):
+        item = LinkedDataParser.parse(response, "ClothingStore")
+        if item:
+            item["website"] = response.url
+            item.update(self.MONSOON)
+            yield item
+            if "Accessorize" in item["name"]:
+                # There is a co-located Accessorize store, this is how they work!
+                item = LinkedDataParser.parse(response, "ClothingStore")
```

Comment:
Do we have to re create `item` and set the website, or can we use the original?

also we probably want `located_in` on this item.

Reply:
You would have to "copy" the original, I have seen some of this behaviour. "deepcopy" I think is more safe / future proof but I am not an expert.

I will set "located_in" but of course that is not quite correct. It should be "located_with" as they many times have two small stores side by side. As you can see I made the comment in the code. I often wonder why people these days write so few comments .... perhaps I start to understand!